ds-collector/ds-collector

```diff
@@ -153,7 +153,10 @@ list_cassandra_nodes() {
     # more accurate disk space check
     node_count="${#cassandraNodes[@]}"
     required_basedir_space="$((${node_count} * 500000))"
-    [ $(df "$baseDir" | tail -n +2 | awk '{print $4}') -ge ${required_basedir_space} ] ||  { echo >&2 "A diagnostic collection of ${node_count} nodes requires at least $((${required_basedir_space} / 1000000))GB free at $baseDir"; exit 1; }
+    # detect if df supports --portability
+    DF_OPT=""
+    ( df --help | grep "\-\-portability" ) && DF_OPT="--portability"
```

Comment:
`-P` would probably be more universal (and also more cryptic)

Reply:
`-P` exists as a different option in other distributions/versions of `df`